src/components/SearchPage/SearchInputComponent.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import TextField from '@material-ui/core/TextField';
+import SearchIcon from '@material-ui/icons/Search';
+import { IconButton, makeStyles } from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+import { useDebouncedCallback } from 'use-debounce/lib';
+import { searchQueryChangeAction } from './SearchPageAction';
+import { useEffect } from 'react';
+import { colors } from 'colors';
+import ClearIcon from '@material-ui/icons/Clear';
+
+const useStyles = makeStyles(() => ({
+  multilineColor: {
+    color: colors.iconColor,
+  },
+  customInput: {
```

Comment:
I think this should be named `textField`, as these styles are applied to the `<TextField>` component.

Reply:
Renamed.